src/CommandingOfficers/Tech.java

```diff
@@ -571,7 +571,7 @@ private static UnitModel createBattleMechModel(GameReadyModels grms)
     BattleMech.name = BATTLEMECH_NAME;
     BattleMech.role = BattleMech.role | UnitModel.SURFACE_TO_AIR;
     BattleMech.costBase = mdTank.costBase*2 + (antiAir.costBase/2);
-    BattleMech.abilityPowerValue = 2.0;
+    BattleMech.abilityPowerValue = 20;
```

Comment:
So with the change in definitions, does that mean this is now worth 1/5 of a star instead of 2 stars? Arguably it should just be worth 0 stars as a free unit; I forget why it was 2 initially.